census_view.cpp

```diff
@@ -1540,9 +1541,73 @@ table_type_converter const& table_type_converter::get_impl()
 
 } // Unnamed namespace.
 
+/// The attribute provider to make all odd rows appear grey.
+
+class CensusViewGridCellAttrProvider
+    :public wxGridCellAttrProvider
+{
+  public:
+    CensusViewGridCellAttrProvider()
+    {
+        lightGray_ = wxColor(0xf7f7f7);
```

Comment:
We shouldn't hardcode this, it will be unreadable with some themes. Please see how `wxDV_ROW_LINES` is handled in `src/generic/datavgen.cpp`.